README.rst

```diff
@@ -107,6 +107,11 @@ Create a repository ``foo``
 Create a new remote ``bar``
 ---------------------------
 
+Docker remotes can be configured with a ``policy``. The default value is ``immediate``. When a
+remote with an ``on_demand`` policy is used to sync a repository, only tags and manifests are
+downloaded. Blobs are only downloaded when they are requested for the first time by a client.
+This can provide significant disk space savings.
+
 ``$ http POST http://localhost:24817/pulp/api/v3/remotes/docker/docker/ name='library/busybox' upstream_name='busybox' url='https://registry-1.docker.io'``
```

Comment:
maybe also update the POST call with the `policy` param?

Reply:
i added it